locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/provider/AbstractJcloudsLocationTest.java

```diff
@@ -48,7 +48,7 @@
 public abstract class AbstractJcloudsLocationTest {
     private static final Logger LOG = LoggerFactory.getLogger(AbstractJcloudsLocationTest.class);
 
-    private final String provider;
+    protected final String provider;
```

Comment:
Why making this field `protected` instead of `private`. Doesn't seem to be used elsewhere, at least in this PR